sv.c

```diff
@@ -12188,15 +12188,15 @@ Perl_sv_vcatpvfn_flags(pTHX_ SV *const sv, const char *const pat, const STRLEN p
             /* the asterisk specified a width */
             {
                 int i = 0;
-                SV *sv = NULL;
+                SV *this_sv = NULL;
```

Comment:
The svs and HEKs in hv.c and mro_core are all pretty generic and generic names are fine*, but here the sv has a specific use, as holding the width of the field, so I think `width_sv` would be better than the generic `this_sv`.

* if they were integers you could probably use `i`, for example